cura/Arranging/GridArrange.py

```diff
@@ -0,0 +1,267 @@
+import math
+from typing import List, TYPE_CHECKING, Optional, Tuple, Set
+
+from UM.Application import Application
+from UM.Math import AxisAlignedBox
+from UM.Math.Vector import Vector
+from UM.Operations.AddSceneNodeOperation import AddSceneNodeOperation
+from UM.Operations.GroupedOperation import GroupedOperation
+from UM.Operations.TranslateOperation import TranslateOperation
+
+
+class GridArrange:
+    offset_x: float = 10
```

Comment:
Why make these class methods?

Reply:
fixed in https://github.com/Ultimaker/Cura/pull/15496/commits/118f49a0520bdd1a3983bbf33b49c3ed8a645fb1